ledger_test.go

```diff
@@ -22,7 +22,7 @@ func TestLogging(t *testing.T) {
 
 			for lvl := FatalLevel; lvl <= DebugLevel; lvl++ {
 				l.write(lvl, msg)
-				Expect(string(mw.Written)).To(Equal(fmt.Sprintf("%s: [%v]\n", lvl, msg)))
+				Expect(string(mw.Written)).To(MatchRegexp(fmt.Sprintf("%s: .*goblin.go:\\d+ \\[%v\\]", lvl, msg)))
```

Comment:
Something seems off with this assertion. Expecting a log line containing `goblin.go:216` seems to defeat the very purpose of this change. Similarly noted for the set of assertions below.